lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1754,6 +1754,11 @@ definitions:
       cloud_subscriberdb_enabled:
         type: boolean
         example: false
+      enable_converged_core:
```

Comment:
I don't like having duplicated data defns. Can you create a separate model defn, then reference it from both locations

Reply:
Sure will do that change.